firefox-ios/Client/Frontend/MicroSurvey/MicroSurveyPrompt.swift

```diff
@@ -0,0 +1,135 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import ComponentLibrary
+
+struct MicroSurveyViewModel {
```

Comment:
nit: could we move it to a file instead of living in the same file, I guess its fine for now considering its to be modified in future

Reply:
moved!